testcontainers-common/src/main/resources/META-INF/spring.factories

```diff
@@ -1,9 +1,6 @@
 org.springframework.cloud.bootstrap.BootstrapConfiguration=\
  com.playtika.test.common.spring.DockerPresenceBootstrapConfiguration
 
-org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
```

Comment:
can you please clarify what is the reason for removing this autoconfiguration?

Reply:
In the deleted class was this comment:

```
//TODO: Drop this workaround after proper fix available https://github.com/spring-cloud/spring-cloud-commons/issues/752
```

So after it was not needed anymore, I deleted the class and the reference here.
I fact, all `destroyMethod = "stop"` Bean attributes are not needed anymore.

Reply:
Can you please revert the changes as the project can still be used with an older version of Spring Cloud. If we remove this workaround users with the older version of Spring Cloud will have issues with containers shutdown. Probably we can update the comment with what I've said.

Reply:
ok